ppsci/data/dataset/cgcnn_dataset.py

```diff
@@ -0,0 +1,320 @@
+from __future__ import print_function, division
+import paddle
+import numpy as np
+from typing import Callable, Dict, Optional, Tuple, Type, Union
+import csv
+import functools
+import json
+import os
+import random
+import warnings
+import pdb
+from paddle import io
+from pymatgen.core.structure import Structure
```

Comment:
pymatgen是一个比较特殊的包，其导入行为请放到try代码中，否则会影响整个项目的正常使用
```suggestion
try
    from pymatgen.core.structure import Structureexcept
except ModuleNotFoundError:
    pass
```

Reply:
已修改